src/server/api/routes/nft/metadata.js

```diff
@@ -0,0 +1,165 @@
+const axios = require('axios');
+const BigNumber = require('bignumber.js');
+const { Router } = require('express');
+
+const { handleRestfulError } = require('../../middleware/error');
+
+const {
+  LIKECOIN_CHAIN_API,
+  LIKECOIN_API_BASE,
+} = require('../../../config/config');
+
+const router = Router();
+
+router.get('/nft/metadata', async (req, res, next) => {
+  try {
+    const { class_id: classId } = req.query;
+    if (!classId) {
+      res.status(400).send('MISSING_CLASS_ID');
+      return;
+    }
+
+    const [
+      [classData, iscnData],
+      ownerInfoRes,
+      listingInfoRes,
+      purchaseInfoRes,
+    ] = await Promise.all([
+      getNFTClassAndISCNMetadata(classId),
+      axios.get(
+        `${LIKECOIN_CHAIN_API}/likechain/likenft/v1/owner?class_id=${classId}`
+      ),
+      axios.get(
+        `${LIKECOIN_CHAIN_API}/likechain/likenft/v1/listings/${classId}`
+      ),
+      axios
+        .get(`${LIKECOIN_API_BASE}/likernft/purchase?class_id=${classId}`)
```

Comment:
should reuse [getNFTPurchaseInfo](https://github.com/nwingt/liker-land/blob/beee54060ac33e3b91d4d96cee1664177e194bd7/src/util/api/index.js#L96)